New package: ReadDBC v1.0.0

JuliaRegistrator opened the pull request:
- Registering package: ReadDBC
- Repository: https://github.com/lego-yaw/ReadDBC.jl
- Created by: @lego-yaw
- Version: v1.0.0
- Commit: b0b93096837e1e85e96eacd329cf7233159d8b62
- Reviewed by: @lego-yaw
- Reference: https://github.com/lego-yaw/ReadDBC.jl/issues/9
- Description: Julia Package for reading DBC (.dbc) files from DATASUS into data frame tables 
<!-- bf0c69308befbd3ccf2cc956ac8a4671113b158fd87a6d4e741b0173dab051a48f7dc43e593ffff4357bc647a3a835983c03ded15d833ca795d626205fd35e1f5683af419f8c554b81bfc5317dfe337fdb722871f8ca2cce5d7877ea11c6a4f27f16bff07448cc67e8420d58a4e3e128b0227ad68b9cf1a06b4d4a138c4ebb4ed544e29b1b58af10f28c1cf767a3056a1fc39bff51908c06fc262c624036e2df0bae67d92b64bbf53cf155fa992e55dc -->

lego-yaw commented:
@JuliaRegistrator register

goerz commented:
This package probably has to be managed with BinaryBuilder. A regular Julia package cannot contain a DLL file.

I also noted in your README:

> Please note that, this package only handles .dbc format used by DATASUS and is not in any way related to handling Microsoft Foxpro or CANdb DBC files.

Since there are apparently other unrelated DBC file formats, that would indicate to me that `ReadDBC` is maybe not a good package name. Is there an alternative that would be more descriptive? Maybe `ReadDatasus` or something along those lines?

JuliaRegistrator committed:
New package: ReadDBC v1.0.0

UUID: 79d50320-df92-4ae6-9b90-d461b5bc1756
Repo: https://github.com/lego-yaw/ReadDBC.jl.git
Tree: 2a828da54ad66f562940f6ca275996298afafdbb

Registrator tree SHA: 17aec322677d9b81cdd6b9b9236b09a3f1374c6a

lego-yaw commented:
@devmotion can you review the changes made so that i can merge

lego-yaw commented:
> This package probably has to be managed with BinaryBuilder. A regular Julia package cannot contain a DLL file.
> 
> I also noted in your README:
> 
> > Please note that, this package only handles .dbc format used by DATASUS and is not in any way related to handling Microsoft Foxpro or CANdb DBC files.
> 
> Since there are apparently other unrelated DBC file formats, that would indicate to me that `ReadDBC` is maybe not a good package name. Is there an alternative that would be more descriptive? Maybe `ReadDatasus` or something along those lines?

@devmotion  i have actually resolved this issue by creating a binarywrapper to handle the .dll issue

devmotion commented:
I'm not sure how I can contribute to this PR, I guess maybe you wanted to ping @goerz?

goerz commented:
Your `src` folder must only contain `.jl` files. And like I said: I think the package name should be more specific

lego-yaw commented:
> I'm not sure how I can contribute to this PR, I guess maybe you wanted to ping @goerz?

Yeah, sorry ...

goerz commented:
Closing in favor of https://github.com/JuliaRegistries/General/pull/114795
